docs/mycroft-technologies/mycroft-core/plugins/tts.md

```diff
@@ -40,17 +40,19 @@ Mycroft will register the `"example_tts"` part in the TTS's `self.config`
 
 ### `get_tts()`
 
-The `get_tts()` method will be called by Mycroft to generate audio and (optionally) phonemes. This is the main method that the plugin creator needs to implement.
+The `get_tts()` method will be called by Mycroft to generate audio and (optionally) phonemes. This is the main method that the plugin creator needs to implement. It is called with:
+- `sentence` (str): a piece of text to turn into audio.
+- `wav_file` (str): path to where Mycroft should store the generated audio data.
 
-It's called with a `sentence`, a piece of text to turn into audio, and a `wav_file` an path to where Mycroft would like to store the generated audio data.
+This method should generate audio data and return a Tuple:
+- path to written data (generally the input argument)
+- viseme list for synthesized audio
```

Comment:
MIght be better to format this as a tuple with the parenthesis and coma to be clear